src/controller/CHIPDeviceController.h

```diff
@@ -65,8 +65,9 @@ namespace chip {
 
 namespace Controller {
 
-constexpr uint16_t kNumMaxActiveDevices = 64;
-constexpr uint16_t kNumMaxPairedDevices = 128;
+constexpr uint16_t kNumMaxActiveDevices    = 64;
+constexpr uint16_t kNumMaxPairedDevices    = 128;
+constexpr uint32_t kAttestationNonceLength = 32;
```

Comment:
This should probably be a `size_t`

Reply:
CSRNonce is using uint32_t for this constexpr. Shall we follow the CSR 'convention' or replace both with size_t?

Reply:
I would argue none of these should be fixed with integers, since the C++ standard way to describe sizes is size_t. if you want to do as in Rome, then do so here, but it will probably mean some casts that could be avoided, elsewhere.

Reply:
changed to size_t